Doc/library/ssl.rst

```diff
@@ -1210,6 +1212,16 @@ SSL sockets also have the following additional methods and attributes:
    .. versionchanged:: 3.9
       IPv6 address strings no longer have a trailing new line.
 
+.. method:: SSLSocket.get_verified_chain()
```

Comment:
put  .. versionadded:: 3.13  tags on these new methods in the docs.

Reply:
Thanks, I will do it.